include/metricq/connection.hpp

```diff
@@ -128,6 +128,7 @@ class Connection
 
     AMQP::Address derive_address(const std::string& address);
 
+    virtual std::string client_version() const;
```

Comment:
Does it make sense to call this `client_version` rather than `version`? Would there be ambiguity with `version`?